rpm/acpid.spec

```diff
@@ -28,13 +20,13 @@ Source9:    acpid-start-script
 Source10:    acpid.start.sh
 Source11:    acpid.service
 Source12:    acpid.conf
-Source100:  acpid.yaml
 Patch0:     acpid-2.0.9-makefile.patch
 Patch1:     acpid-2.0.14-bugfix-incorrect-sizeof-usage-for-memset.patch
 Requires:   systemd
 Requires(preun): systemd
 Requires(post): systemd
 Requires(postun): systemd
+BuildRequires:  pkgconfig(systemd)
```

Comment:
Why is this needed?

Reply:
Macros like %{_unitdir} are provided by the systemd package.